### DISCUSSION TITLE
When is the next release scheduled?

### DISCUSSION THREAD

**Discussion by chnsh** [alshedivat/al-folio#2481] [Q&A] [answered]
I would love to use the new features and it looks like it's been over 5 months since the previous release. I just wanted to check about the schedule.

**Comment by george-gca** [comment]
We are currently not following a strict release schedule, but the tip of the master is stable enough to be used as the current version. But I agree that it is about time we fix a new release. What do you think @alshedivat?

**Comment by alshedivat** [comment, marked as answer]
my apologies, I've been delinquent on tagging new releases on time. will tag the next one on Jul 1st, and will aim to release subsequent versions on quarterly basis (Oct 1st, Jan 1st, Apr 1st, etc.).
